pkg/kubectl/resource/helper.go

```diff
@@ -131,14 +132,12 @@ func (m *Helper) Create(namespace string, modify bool, obj runtime.Object) (runt
 func (m *Helper) createResource(c RESTClient, resource, namespace string, obj runtime.Object) (runtime.Object, error) {
 	return c.Post().NamespaceIfScoped(namespace, m.NamespaceScoped).Resource(resource).Body(obj).Do().Get()
 }
-func (m *Helper) Patch(namespace, name string, pt types.PatchType, data []byte) (runtime.Object, error) {
+func (m *Helper) Patch(namespace, name string, pt types.PatchType, data []byte) *client.Request {
 	return m.RESTClient.Patch(pt).
 		NamespaceIfScoped(namespace, m.NamespaceScoped).
 		Resource(m.Resource).
 		Name(name).
-		Body(data).
-		Do().
-		Get()
+		Body(data)
```

Comment:
I don't quite understand why moving `.Do().Get()` out.

Reply:
Helper was never intended to accept large numbers of arguments.  It only exists to abstract the default setup for a request.  We need to start removing parameters from helper.